frontend/src/ui/metadata.tsx

```diff
@@ -68,14 +68,15 @@ type DescriptionProps = {
  * proper paragraphs, but not a full markdown renderer by a longshot.
  */
 export const Description: React.FC<DescriptionProps> = ({ text, className }) => {
-    if (text === null) {
-        return null;
+    const { t } = useTranslation();
+
+    if (text === null || text.trim() === "") {
+        return <div {...{ className }} css={{ fontStyle: "italic" }}>
+            {t("manage.my-videos.no-description")}
+        </div>;
     }
 
     const stripped = text.trim();
```

Comment:
Why not lead with something like

```javascript
const stripped = text?.trim();
```

and then the condition can just be `!stripped` saving the double trimming.